update jersey to version 2

chrisdoc opened the pull request:
add gitignore
update jersey to version 2
updating WNSClient to use jersey 2 and jackson 2
add junit 4 tests
add gradle support
cleanup

fernandospr commented:
Please, modify the pom from 1.4-SNAPSHOT to 1.3.1-SNAPSHOT as this would not be a release with new features, these are internal changes.

chrisdoc commented:
Hey @fernandospr thank you for your comments. I removed the test and change the instance of to getType(). I also changed the version number however I did not change it to 1.4-SNAPSHOT in the first place. This is already been set in the tip of master

fernandospr commented on pom.xml:
```diff
@@ -3,7 +3,7 @@
 	<artifactId>java-wns</artifactId>
 	<packaging>jar</packaging>
-	<version>1.4-SNAPSHOT</version>
+	<version>1.3.1</version>
```
This must be 1.3.1-SNAPSHOT because this is still under development.

fernandospr commented on build.gradle:
```diff
@@ -0,0 +1,31 @@
+apply plugin: 'java'
+
+group = 'com.github.fernandospr'
+version = '1.3.1'
```
This must be 1.3.1-SNAPSHOT because this is still under development.

chrisdoc commented:
PING

fernandospr commented:
Sorry @chrisdoc for the delay, I have decided not to use Gradle. 
I don't plan to mantain the maven and gradle files.
For now, I would like to stick to maven.
If you want you can fork this repository and use gradle only.

chrisdoc commented:
@fernandospr no problem, I removed the gradle stuff. I just feel more comfy with Gradle then Maven :smile:

fernandospr commented:
@chrisdoc People are getting problems when I merged your PR... It seems that it is not serializing the list of objects the way it used to be. Please check issue: [12](https://github.com/fernandospr/java-wns/issues/12)
Please can you help me solve the issue?
